code-challenges.js

```diff
@@ -18,9 +37,17 @@ var cubeAndSum1 = [2, 3, 4]
 var cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
 
+// declare function cubeNumThenSUm to take an array as argument
+const cubeNumThenSum = (array) => {
+    let totalSum = 0
+    // first map through array and cube all numbers
+    // reduce the array to get the sum
+    let cubedThenSumArray = array.map(value => value ** 3).reduce((accumulator, currentValue) => accumulator + currentValue)
+    return cubedThenSumArray
+}
 
```

Comment:
Love the reduce. It is a great method. 
I see what you are saying with being concise. I like that you are exploring different approaches to code.